assets/src/block-templates/side-image-with-text-and-cta/template.js

```diff
@@ -0,0 +1,28 @@
+import mainThemeUrl from '../main-theme-url';
+
+const { __ } = wp.i18n;
+
+const template = ({
+  title = '',
+  titlePlaceholder = __('Enter title', 'planet4-blocks-backend'),
```

Comment:
I'm not sure we need this attribute 🤔 I feel like the placeholder will always be the same